template-for-repository/proofs/Makefile.common

```diff
@@ -271,6 +271,26 @@ PROOF_SOURCES ?=
 # report, making the proof run appear to "hang".
 CBMC_TIMEOUT ?= 21600
 
+# Proof writers could add function contracts or loop invariants in
+# CBMC proofs. They should follow three distinct processes:
+# 1. Check whether a function contract is sound, where CHECK_CONTRACTS
+#    contains the list of functions with contracts to be checked.
+# 2. Replace function calls with their correspondent function contracts,
+#    where REPLACE_WITH_CONTRACTS contains the list of functions with
+#    contracts to be replaced. One must check separately whether a function
+#    contract is sound before replacing it.
+# 3. Apply loop invariants during verification, where APPLY_LOOP_INVARIANTS
+#    contains the list of functions with loop invariants to be applied.
+#    This option is currently unavailable.
+CHECK_CONTRACTS ?= ""
+CHECK_CONTRACTS_SET := $(patsubst %,--enforce-contract %, $(CHECK_CONTRACTS))
+
+REPLACE_WITH_CONTRACTS ?= ""
+REPLACE_CONTRACTS_SET := $(patsubst %,--replace-call-with-contract %, $(REPLACE_WITH_CONTRACTS))
```

Comment:
[Nitpick]

@mww-aws  @markrtuttle Should we use `REPLACE_WITH_CONTRACTS` or `USE_CONTRACTS`? I am fine with either.

Reply:
I adopted `USE_FUNCTION_CONTRACTS` following the same name scheme as the other variables.